### PR TITLE
Add template for non-licensed admin users

### PR DESCRIPTION
**Pull request by srajko**

### CODE REVIEW

**Comment by johndavidsparrow** [comment]
@sarabreeding @tristalou

File: src/emails/organizationInviteNewAdminEmail.hbs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+description: Sent to a non-licensed user with no GitKraken account after being invited to use GitKraken
+layout: default.hbs
+subject: You've been invited to manage %PLAN_NAME%!
```
Comment (sarabreeding) [inline src/emails/organizationInviteNewAdminEmail.hbs]: You've been invited to manage a %PLAN_NAME% account!

File: src/emails/organizationInviteNewAdminEmail.hbs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+header: You've been invited to manage the %PLAN_NAME% account with %ORG_NAME%
```
Comment (sarabreeding) [inline src/emails/organizationInviteNewAdminEmail.hbs]: You've been invited to manage %ORG_NAME%'s %PLAN_NAME% account.

File: src/emails/organizationInviteNewAdminEmail.hbs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+<p class='align-center'>
+  <a href='%LINK_URL%'>Register</a> your account to begin managing %PLAN_NAME%.
```
Comment (sarabreeding) [inline src/emails/organizationInviteNewAdminEmail.hbs]: Register your account to begin managing.

**Comment by srajko** [comment]
Fixed text - thanks @sarabreeding!

**Pull request merged**